db/migrate/20220121111127_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :authorid
```

Comment:
This attribute is not required. It will be nice to remove it. Instead, it will be added as a foreign_key.

Reply:
This issue is well resolved! ✔️